wcag20/sources/understanding/navigation-mechanisms-skip.xml

```diff
@@ -6,35 +6,34 @@
    <div3 role="intent">
       <head/>
 
-<p>この達成基準の意図は、コンテンツ内を一つずつ順を追って行き来している利用者がウェブページのメインコンテンツへ直接移動できるようにすることである。ウェブページ及びウェブアプリケーションには、他のページ又は画面でも現れるコンテンツがしばしばある。コンテンツの中で繰り返されているブロックの例としては、ナビゲーションリンク、見出しのグラフィック、そして広告を表示するフレームなどが挙げられる。個々の単語、フレーズ、又は単独のリンクなどの小さな繰り返される部分は、この達成基準の趣旨においては、ブロックとしてみなされない。</p>
-<p>このことは、画面を見ている利用者が、画面の中央（通常は、メインコンテンツがあるところ）に意識を集中することによって、繰り返しているコンテンツを無視することができることや、マウスを使用している利用者が、目的のリンクの前にあるすべてのリンク又はフォームのコントロールと出くわさずに、1 回のクリックでそのリンクを選択できることと対照的である。</p>
-<p>この達成基準の意図は、コンテンツ制作者にユーザエージェントが提供している機能と重複する手段を提供するように求めることではない。ほとんどのウェブブラウザは、利用者がフォーカスをページの先頭に戻すことのできるキーボードショートカットを提供しているので、ナビゲーションリンク一式をウェブページの下部で提供している場合は、いわゆる「スキップ」リンクを提供する必要はないかもしれない。</p>
+<p>この達成基準の意図は、コンテンツを順番にナビゲートする人にウェブページのメインコンテンツへより直接的なアクセスをできるようにすることである。ウェブページ及びウェブアプリケーションには、他のページ又は画面でも現れるコンテンツがしばしばある。コンテンツの中で繰り返されているブロックの例には、ナビゲーションリンク、見出しのグラフィック、及び広告のフレームなどを含むが、これに限定されるものではない。個々の単語、フレーズ、又は単独のリンクなどの小さな繰り返されるセクションは、この達成基準が意図するブロックとはみなされない。</p>
+<p>これは、目の見える利用者に画面の中央 (通常はメインコンテンツが現れるところ) に注目することで繰り返される要素を無視する能力があることや、マウスの利用者に選択したい要素より前にある全てのリンクやフォームコントロールに遭遇することなく、マウスクリック 1 回でリンクを選択する能力があることとは、対照的なものである。</p>
```

Comment:
either ... or ～ ではないeitherだとどう訳出すればいいのか？という疑問が（原文が怪しい？）。

＞能力があることや
能力があること又は、

＞全ての
このeveryを全てのとするのは違和感が。

＞することなく
これはrather thanから離れすぎでは。

＞リンクやフォームコントロール
リンクもしくはフォームコントロール

Reply:
ここは正直ちょっと原文が読みにくすぎてどうしようもない感があります。

either の後ろに or はありますが、形が揃っていないので変ではありますね。
気持ち的には either by focusing or by using mouse... と書くつもりだったのでしょうか。そう考えると、
(目の見える利用者が注目する) or (マウスの利用者が選択する)
ではなく、
(目の見える利用者が、 (注目する) or (マウスを使って選択する))
なのかもしれませんね。どちらでも意味は通じます。
後者の解釈だとこうとか?

→これは、目の見える利用者が画面の中央 (通常はメインコンテンツが現れるところ) に注目して繰り返される要素を無視したり、マウスを使って選択したい要素より前にあるリンクもしくはフォームコントロール全てに出くわすのでなくマウスクリック 1 回でリンクを選択する能力があることとは、対照的なものである。